source/cloud-native-platform/path-to-live/public-dns.html.md.erb

```diff
@@ -17,32 +17,24 @@ follow the [shuttering](shutter.html#pre-requisite) instructions for which recor
 for a new service.
 
 ## Setup required for an application
-- Copy the FrontDoor endpoint hostname for the environment:
-
-<%= partial 'front-door-endpoints' %>
-
 - Clone the [Azure Public DNS](https://github.com/hmcts/azure-public-dns) repository.
 - Under environments directory there are yaml files depicting environments that represents zones corresponding to each environment.
 There can be multiple zones in one environment,
-- Create afdverify CNAME records in the appropriate zone file for environments.
-Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull/1387/files),
+- Create TXT records in the appropriate zone file for environments.
+Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull/1409/files),
 - Create a PR with records as per the below convention
 (make sure you replace the variables with the correct values).
 
 
 ```
-- name: "afdverify.${name}"
-  ttl: 300
-  record: "afdverify.{copy endpoint hostname from frontdoor}"
-- name: "${name}"
```

Comment:
this is still needed? but you may want to split it into multiple steps?

Reply:
added